content/cloud/kv-cloud-tutorial.md

```diff
@@ -0,0 +1,138 @@
+
+> Please check that you have [Spin CLI](../spin/v2/cli-reference.md) v2.0 or greater, and [`cloud` plugin](https://github.com/fermyon/cloud-plugin) v0.7.0 or greater installed. 
+
+## Creating a New Spin Application
+
+If you already have a Spin application, you can skip this step. If you do not have a Spin application locally you will need to [create one](/spin/quickstart#create-your-first-application).
+
+## Grant Key Value Store Permission
+
+To tell Spin that we want to use a Key Value Store, we only need to [grant Key Value Store permission to a component in the application’s manifest](/spin/kv-store-api-guide#granting-key-value-store-permissions-to-components) (the `spin.toml` file) by supplying a [label](linking-applications-to-resources-using-labels.md). For example:
+
+```
+[component.example]
+key_value_stores = ["default"]
+```
+
+The new database file (`sqlite_key_value.db`) is created in your application's `.spin` folder when you run this Spin application. The database persists across Spin application invocations and updates. The key value store is empty to start.
+
+## Creating a Key Value Store in Fermyon Cloud
+
+You have two options to create your key value store (both are equally valid):
+* Option A: Implicitly create a key value store (with an automatically assigned arbitrary database name) via `spin cloud deploy`. For example `inspirational-pig` or similarly named database will be created.
+* Option B : Explicitly provide the name of the database to create via the `spin cloud kv create` command. For example, `spin cloud kv create finance-database` creates a database called `finance-database` inside your Fermyon Cloud account.
+
+Please run whatever step is relevant to your needs.
+
+<!-- markdownlint-disable-next-line titlecase-rule -->
+### Implicitly Create A Key Value Store Via `spin cloud deploy`
+
+When you first `spin cloud deploy` your application (with the `key_value_stores` configuration, as shown above), Fermyon Cloud will create a new database for your Cloud application.
+
+The new database is initially empty, so you will need to create tables (and potentially initial data) before it is useful.  To do this:
+
+1. Find the name of your Cloud database. To do this, run the following command:
```

Comment:
There is a lot of database in here. Might need to do a ctrl F just to scrub it out
```suggestion
1. Find the name of your Cloud key value store. To do this, run the following command:
```